tests/utils.go

```diff
@@ -2106,7 +2103,9 @@ func GetGuestAgentUserData() string {
                 `, guestAgentUrl, guestAgentUrl, GetUrl(StressHttpUrl))
 }
 
-func GetIPv6NetworkData(ipAddress string, gateway string, dnsServer string, searchDomains []string) string {
+// Returns NetworkData for configuring a dynamic IPv4 address, and a static
+//IPv6 address, along with DNS configuration.
+func GetCloudInitNetworkData(ipAddress string, gateway string, dnsServer string, searchDomains []string) string {
```

Comment:
@EdDev I'm renaming this function, since it no longer is ipv6 specific. Is this name OK ?

Reply:
Seems fine.
The `gateway` may be confusing, as it is 6 only... but I think this needs to be generalized anyway so we can adjust it then.
The comment should be enough for now.